js/jsmodules.go

```diff
@@ -50,6 +51,10 @@ func getInternalJSModules() map[string]interface{} {
 		"k6/http":                 http.New(),
 		"k6/metrics":              metrics.New(),
 		"k6/ws":                   ws.New(),
+		"k6/experimental/grpc": newRemovedModule(
+			"k6/experimental/grpc has been removed, please use k6/net/grpc instead." +
```

Comment:
I would prefer if the message makes obvious that the same functionality is now available in `k6/net/grpc`.

```suggestion
			"k6/experimental/grpc has been graduated, please use k6/net/grpc instead." +
```
or _merged_